src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -693,15 +695,17 @@ This resets the cluster to its initial state.`,
 		}),
 	}
 	portForward.Flags().Uint16VarP(&port, "port", "p", 30650, "The local port to bind pachd to.")
-	portForward.Flags().Uint16Var(&remotePort, "remote-port", 650, "The remote port that pachd is bound to in the cluster.")
+	portForward.Flags().Uint16Var(&remotePort, "remote-port", 1650, "The remote port that pachd is bound to in the cluster.")
```

Comment:
Support configuring the S3 and PFS HTTP container ports in port-forwarding, these were hard-coded.